xbmc/pvr/recordings/PVRRecordingsPath.h

```diff
@@ -56,10 +60,12 @@ namespace PVR
 
   private:
     static std::string TrimSlashes(const std::string &strString);
+    size_t GetDirectoryPathStart() const;
 
```

Comment:
not really happy with such method names. you should definitely add a method doc which makes it clear what does it return.